app/js/components/manageApps/SingleAddon.jsx

```diff
@@ -66,16 +66,6 @@ export default class SingleAddon extends React.Component {
 
     return (
       <tr key={key}>
-        <td>
-          <img
-            src={app.appDetails &&
-              app.appDetails.icon !== null &&
```

Comment:
What is the reason behind removing this?

Reply:
You said i should remove the first column on the addon manager. That is the logo column.